sycl/plugins/level_zero/usm_allocator.cpp

```diff
@@ -242,12 +337,15 @@ std::ostream &operator<<(std::ostream &Os, const Slab &Slab) {
 }
 
 Slab::Slab(Bucket &Bkt)
-    : MemPtr(Bkt.getMemHandle().allocate(settings::SlabSize)),
+    : // MemPtr(Bkt.getMemHandle().allocate(Bkt.getSize())),
       // In case if bucket size is not that SlabSize % b.getSize() == 0, we
       // would have some padding at the end of the slab.
       Chunks(settings::SlabSize / Bkt.getSize()), NumAllocated{0},
       bucket(Bkt), SlabListIter{}, FirstFreeChunkIdx{0} {
-
+  size_t SlabAllocSize = Bkt.getSize();
+  if (SlabAllocSize < settings::SlabSize)
```

Comment:
Should you name it SlabMinSize, then?

Reply:
Not really. Here we are choosing the size to allocate, hence the name SlabAllocSize. The slab minimum size is pre-determined to be settings::SlabSize, i.e. 64KB. We allocate at least that.

Reply:
maybe name `settings::SlabMinSize`?

Reply:
Yes